angular/src/app/screens/auth/screens/book-list/book-list.component.html

```diff
@@ -1 +1,19 @@
-<p>book-list works!</p>
+<div class="book-search">
+  <input class="book-search-input" name="inputSearch" role="search" [(ngModel)]="inputSearch" type="search"
+    placeholder="Buscá por título del libro..." /><img class="book-icon-search" src="assets/search-icon.png"
+    alt="Search icon" />
+</div>
+
+<div class="books-container">
+  <ng-container *ngIf="(books | filterBooks: inputSearch).length === 0; else elseTemplate">
```

Comment:
ngIf logic should be on component file

Reply:
Also, elseTemplate should be the worse case scenario. So I think we could change condition 👍🏻